test/client-test.js

```diff
@@ -192,5 +192,154 @@ describe('Client', () => {
 
       await this.fhirClient.read({ resourceType: 'Patient', id: 'test-access-token' });
     });
+
+    describe('#create', () => {
+      it('returns a successful operation outcome', async function () {
+        const newPatient = {
+          resourceType: 'Patient',
+          active: true,
+          name: [{ use: 'official', family: ['Coleman'], given: ['Lisa', 'P.'] }],
+          gender: 'female',
+          birthDate: '1948-04-14',
+        };
+
+        nock(this.baseUrl)
+          .matchHeader('accept', 'application/json+fhir')
+          .post('/Patient')
+          .reply(201, () => readStreamFor('patient-created.json'));
+
+        const response = await this.fhirClient.create({
+          resourceType: newPatient.resourceType,
+          resource: newPatient,
+        });
+
+        expect(response.resourceType).to.deep.equal('OperationOutcome');
```

Comment:
I think we can just do `to.equal('OperationOutcome')` here and throughout tests-- usually just need deep equality for objects